args.json

```diff
@@ -16,5 +16,5 @@
     "chain_name": "testChain",
     "custom_subnet_vm_path": "",
     "custom_subnet_vm_url": "",
-    "subnet_genesis_json": "github.com/kurtosis-tech/avalanche-package/static_files/genesis.json"
-}
\ No newline at end of file
+    "subnet_genesis_json": "/static_files/genesis.json"
```

Comment:
do we want to suggest `./` or `/`?

Reply:
with `/` is relative to the package root, it can be used in any file this way